xla/service/gpu/transforms/cudnn_custom_call_compiler.cc

```diff
@@ -175,15 +177,21 @@ absl::StatusOr<se::gpu::CudnnGraph> BuildGraphForCustomCallToForwardFMHA(
         page_table_v,
         TensorDescriptorFor(custom_call->operand(input_index++)->shape()));
   }
-  TF_RET_CHECK(input_index == custom_call->operand_count());
 
+  auto computations = custom_call->called_computations();
+  const HloComputation *score_mod_fwd_comp = nullptr;
+  if (computations.size() == 1) {
+    score_mod_fwd_comp = computations[0];
+    input_index += score_mod_fwd_comp->num_parameters() - 1;
+  }
```

Comment:
I guess computations.size() > 1 would be an error? Could you please add a TF_RET_CHECK for that?

Reply:
I think at most 1 computation will be attached here because of this: https://github.com/openxla/xla/blob/74466a328188c70b5be64153aedb93516c5dd11b/xla/hlo/translate/mhlo_to_hlo/mlir_hlo_to_hlo.cc#L2560. Should we still check here?